packages/react-composites/src/composites/MeetingComposite/adapter/AzureCommunicationMeetingAdapter.ts

```diff
@@ -67,13 +67,14 @@ export class AzureCommunicationMeetingAdapter implements MeetingAdapter {
   }
 
   public joinMeeting(microphoneOn?: boolean): void {
-    throw new Error('Method not implemented.');
+    this.callAdapter.joinCall(microphoneOn);
   }
-  public async leaveMeeting(forEveryone?: boolean): Promise<void> {
-    throw new Error('Method not implemented.');
+  public async leaveMeeting(): Promise<void> {
+    await this.chatAdapter.removeParticipant(this.chatAdapter.getState().userId);
```

Comment:
Both chat and calling state save userId. Should we hoist a single `userId` to the meeting adapter state directly instead of the duplicate values?

Reply:
We do have one at the MeetingAdapterClientState level (where displayName etc is stored). However here we use the chat one is simply to get around the fact that chat doesn't have an easy leaveChat() fn and instead requires you to remove yourself so was the simplest place to grab that user id.

If you wanted the meetingAdapter to also have this ID I could update the ctor to be:
```
constructor(userId: string, callAdapter: CallAdapter, chatAdapter: ChatAdapter) {
  this.userId = userId;
}
```

But pushing back a bit just to make the code simpler - happy to update though if you feel strongly